sql_user_check_mysql/database_stats.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+port="3306"
+mode="stats"
+
+while getopts h:u:m:o:p flag
+do
+    case "${flag}" in
+        h) dbhost=${OPTARG};;
+        u) username=${OPTARG};;
+        m) mode=${OPTARG};;
+        o) outputfile=${OPTARG};;
+        p) port=${OPTARG};;
+    esac
+done
+
+
+echo "dbhost: $dbhost";
+echo "username: $username";
+echo "port: $port";
+echo "mode: $mode";
+echo "outputfile: $outputfile";
```

Comment:
No need to print this. If you do want to print it do it in one line with proper message

Reply:
printing in one line